environments/common.yaml

```diff
@@ -164,7 +164,25 @@ jobs:
       // Filter by name (with regular expression)
       headRegexFilter {
         // Regular expression
-        regex('main|PR-.*')
```

Comment:
Aren't we losing the PRs built in the non-release jobs using this `mainHeadRegexFilter` trait?